client/my-sites/sidebar/index.jsx

```diff
@@ -183,7 +194,14 @@ export class MySitesSidebar extends Component {
 				label={ label }
 				selected={ itemLinkMatches( [ '/home' ], path ) }
 				link={ '/home' + siteSuffix }
-			/>
+			>
+				{ ! hideTaskProgress && (
+					<div className="sidebar__checklist-progress">
+						<p className="sidebar__checklist-progress-text">{ translate( 'Finish setup' ) }</p>
+						<ProgressBar value={ 2 } total={ taskCount } />
```

Comment:
Trying to figure out what `value` should be here by calculating the number of completed tasks is proving to be the problem! I'm going to try to figure it out again, but haven't had much luck so far.